hardware/Netatmo.cpp

```diff
@@ -2561,7 +2585,9 @@ bool CNetatmo::ParseHomeStatus(const std::string& sResult, Json::Value& root, st
 						b << ";";
 						b << boiler_boost;
 						std::string sValue = b.str().c_str();
+						const double Level = module["boiler_valve_comfort_boost"].asDouble();
 						//UpdateValueInt(0, ID.c_str(), 0, pTypeGeneralSwitch, sSwitchGeneralSwitch, '0', 255, '0', sValue.c_str(), bName,  bIsActive, m_Name);
+						//SendSwitch(crcId, 9, batteryLevel, bIsActive, Level, bName, m_Name, mrf_status);
```

Comment:
@JurgenLB , add a line but commented out? Remove it?

Reply:
Why add a line and comment it out? Adding code that is not used does not help.
Remove it? (And the variable created above as well)